### ISSUE TITLE
Open project popup keeps appearing

### ISSUE THREAD

**Issue by regularlabs** [Ultimaker/Cura#5469]
Using Cura 4 on OSX.

Every time I open a Cura project (.3mf), I get this popup:
![image](https://user-images.githubusercontent.com/13553242/54741357-817f8c00-4bbe-11e9-9e23-bb0c2247a132.png)

As you see, the 'Remember my choice' option is checked. However, it seems to not do anything with that info. I keep getting that confirm popup.

Also in the settings, this option is saved correctly (but ignored):
![image](https://user-images.githubusercontent.com/13553242/54741453-cf948f80-4bbe-11e9-886c-00fdb10c1e26.png)

And a connected other thing:
After clicking open, the 'Project Summery' is shown:
![image](https://user-images.githubusercontent.com/13553242/54741482-ef2bb800-4bbe-11e9-986f-0f2b10ac46c3.png)

Can we disable that? As it just adds another irritating, unnecessary popup and click to the process.

**Comment by Vandrasc** [comment]
Hi @regularlabs , first of all, thank you for your feedback!

Second, I am not able to reproduce the first issue, once checked the pop up doesn't appear for me anymore. Can you uncheck it and check it again? Maybe it is not updating correctly.

For the Open Project summary, I can bring it up with the team and maybe we can add an option in Preference where you can enable and disable it. We consciously decided to display it because it may override your current configuration.

**Comment by regularlabs** [comment]
Ok, thanks.
Unchecking it in the dialog did nothing. It didn't even changelog the setting in the preferences.
I also rest that setting. And rechecked the option after restarting. But the dialog keeps showing everytime I open a project.

Please note: I open the project when Cura id not running. So double-clicking on the file, which starts up Cura.

Once Cura is running, the setting is working.

**Comment by regularlabs** [comment]
Oh, scratch that last bit. Once Cura is running, the dialog appears too every time!

**Comment by Ghostkeeper** [comment]
I'm also unable to reproduce this issue. It doesn't ask me again whether to open as project or import models after I checked the checkbox.

Do other preferences get saved properly after restarting Cura?

It does still ask whether to override the current profiles or create new profiles for them if I have any profiles in Cura that match the IDs of the profiles in the 3MF file, but have different settings. We discussed this, and we decided not to solve this. There are several problems with not showing that dialogue:
* The dialogue shows a choice for three profile layers. You could have a preference for each of them which is a bit esoteric, or you could have one preference that decides on all of them. But then that preference needs to be set to "ask first" before you get the choice for any of them. You can't "ask first but only if the material is different" then.
* If you click to override without asking, you'll get data loss without warnings, which is dangerous for professional users.
* If you click to always override without asking, you'll get hundreds of profiles at some point, which is not what anyone wants and slows Cura down.
* It's not worth the effort to find solutions for that.

**Comment by regularlabs** [comment]
> Do other preferences get saved properly after restarting Cura?
The setting is saved. It just doesn't affect the behaviour.

> If you click to always override without asking, you'll get hundreds of profiles at some point, which is not what anyone wants and slows Cura down.
There are 2 options for the 3 settings: "Update" and "Create New". I guess you are referring to "Create New" when you say "override"?

Of cause, when hiding that summery, it would simply update the settings.
This doesn't save anything, it just changes the setting values, like what would happen if you would do this manually.

> If you click to override without asking, you'll get data loss without warnings, which is dangerous for professional users.
Then professional users shouldn't disable that dialogue!

**Comment by Ghostkeeper** [comment]
> The setting is saved. It just doesn't affect the behaviour.

Sorry, let me rephrase: Do other settings get properly read after restarting Cura?

**Comment by regularlabs** [comment]
> The first gets skipped when you set the preference.
That's the issue. It never gets skipped.

See below screencast:
![cura](https://user-images.githubusercontent.com/13553242/54997504-225db500-4fcc-11e9-89a9-d8023b64096b.gif)

**Comment by Ghostkeeper** [comment]
I'm not able to reproduce that here. Here's what I'm seeing: http://dulek.net/work/saving_3mf_preference.ogv

Funny how the checkbox is enabled by default on your screen.

**Comment by regularlabs** [comment]
Yeah, but you're not on Mac OSX.

**Comment by Vandrasc** [comment]
I am using OS X and unfortunately, I still cannot reproduce. Did you have this issue with a previous version as well? 4.0 Beta or even 3.6? 

I am running Mojave 10.14 btw.

**Comment by regularlabs** [comment]
Not sure if I had the issue on 3.6. Haven't used 4beta.
I'm on High Sierra 10.13.
